generator/internal/scaffold_generator.h

```diff
@@ -74,6 +74,10 @@ void GenerateQuickstartBuild(
     std::ostream& os, std::map<std::string, std::string> const& variables);
 void GenerateQuickstartBazelrc(
     std::ostream& os, std::map<std::string, std::string> const& variables);
+void GenerateSamplesBuild(std::ostream& os,
```

Comment:
s/Build/Bazel/ perhaps??

Reply:
I will let it be. We already have a `GenerateQuickstartBuild()` function, and calling that `GenerateQuickstartBazel()` would be confusing considering we also have `GenerateQuickstartWorkspace()`.